react-form-controls/my-app/src/RegistrationFormControlled.js

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react';
+export default function RegistrationFormControlled() {
+
+const [username, setUsername] = useState('');
+const [password, setPassword] = useState('');
+
+  function handleSubmit(e) {
+    e.preventDefault();
+    console.log(username, password);
+  }
+
+  return (
+    <form
+      onSubmit={handleSubmit}
+    >
+
+      <label>Username
+        <input type="text" name="username" value={username} onChange={(e) => setUsername(e.target.value)}></input>
```

Comment:
For controlled components, the `name` attribute is optional and doesn't serve a useful purpose.

Reply:
Okay, that makes sense.  Good to know.